spec/features/admin/structure_spec.rb

```diff
@@ -21,18 +21,27 @@
     end
     before { visit admin_structure_path(structure) }
 
-    it do
-      expect(page).to have_content 'Ma structure'
-    end
+    describe 'Ma structure' do
+      it do
+        expect(page).to have_content structure.nom
+      end
```

Comment:
On peut mettre ca en une ligne : 
```suggestion
      it { expect(page).to have_content structure.nom }
```